src/stepper_interface.cpp

```diff
@@ -18,28 +35,25 @@ uint8_t MotorController::StepperInterface::getStepperWires() const {
 }
 
 void MotorController::StepperInterface::setStepperWires(const uint8_t newAlmount) {
-    if (newAlmount >= 3 && newAlmount <= 5) {
+    if (newAlmount == 2) {
+        stepperWires = newAlmount;
+    } else if (newAlmount == 4) {

```

Comment:
You could combine the 4 and 5 in a single statement: else if (newAlmount  == 4 || 5){...}; since the functionality is the same anyway. Also, 'almount'  seems to be a typo. An innocent one, but if you're going to fix this, you might as well fix that.